Fix travis

linuxbandit opened the pull request.

linuxbandit committed:
(fix) First attempt to make tests working

linuxbandit committed:
(fix) forgot secret generation, wrong param name, add notes to self

serge1peshcoff commented on .travis.yml:
```diff
@@ -8,17 +8,10 @@ services:
-    - bash ./oms.sh exec -T -d omsserviceregistry true 2>/dev/null 
-    - bash ./tests/LocalLoopbackTest/LocalLoopbackTest.sh
+    - bash ./helper.sh --execute -- -T -d traefik help 2>/dev/null 
+    - bash ./helper.sh --execute -- -T -d oms-frontend true 2>/dev/null 
```
can you also put oms-events and oms-statutory there?
also keep in mind that oms-frontend is not meant to be run forever, it just builds stuff and exits, so there can be cases where it's not running and the command is executed against it, causing the test to fail

linuxbandit replied:
honestly i don't even know if this is a good way of testing them, i translated how it was to how it should be now - and at least a forgotten definition of secrets (e.g., taking the example of the last break that happened), or the lack of a folder (that you fixed) would have been spotted

serge1peshcoff reviewed (approved):
I approve generally

linuxbandit committed:
(impr) add core, statutory, events container

linuxbandit committed:
(fix) remove mysterious lines that break

Also fix Makefile syntax (missing tab)
The .travis.yml will be improved another time, for now it's
   already a great improvement
